test/endToEnd/endToEndTests.ts

```diff
@@ -1,8 +1,14 @@
+import { resetResult } from '../../src/core/result';
 import { run } from '../../src/core/runner';
+import GoodFencesError from '../../src/types/GoodFencesError';
 import GoodFencesResult from '../../src/types/GoodFencesResult';
 import normalizePath from '../../src/utils/normalizePath';
 
 describe('runner', () => {
+    afterEach(() => {
+        resetResult();

```

Comment:
@Adjective-Object Can you think of any reason we wouldn't want to do this at the very end of the runner?  That way it would apply to any consumer of good-fences, not just the tests.  (You'd have to fix `resetResult` to create a new instance of the result object rather than mutating the existing one.)